cmd/generate-analytics-api.rb

```diff
@@ -41,19 +41,19 @@ def analytics_json_template(category_name, data_source: nil)
   end
 
   def run_formula_analytics(*args)
-    # Give InfluxDB some breathing room.
-    sleep 4
-
     puts "brew formula-analytics #{args.join(" ")}"
 
     retries = 0
     result = system_command HOMEBREW_BREW_FILE, args: ["formula-analytics", *args], print_stderr: false
 
     while !result.success? && retries < MAX_RETRIES
-      # Give InfluxDB some more breathing room.
-      sleep 4**(retries+2)
+      $stderr.puts(result.stderr)
+
+      # Give InfluxDB some breathing room.
+      sleep 2**(retries+2)
```

Comment:
```suggestion
      sleep 4**(retries+2)
```

let's keep the backoff more aggressive